### DISCUSSION TITLE
MQTT Explorer does not start on macOS / ARM

### DISCUSSION THREAD

**Discussion by ulrichwinter** [thomasnordquist/MQTT-Explorer#833] [Q&A] [answered]
After downloading the newest macos / ARM version from [here](https://github.com/thomasnordquist/MQTT-Explorer/releases/download/v0.4.0-beta.6/MQTT-Explorer-0.4.0-beta.6-arm64.dmg) it does not start, even following this [instructions](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac)

Press Control+MousClick > Open
leads to this error message:
<img width="296" alt="image" src="https://github.com/user-attachments/assets/1bde391e-1e75-4e54-b2a1-f3fa442ac7b5">

I also tried installing it with homebrew and got a hint, that mqtt-explorer is compiled for intel only and would require Rosetta 2, which I don't have installed.

**Comment by ulrichwinter** [comment, marked as answer]
The solution is: Install MQTT explorer from the mac [AppStore](https://apps.apple.com/us/app/mqtt-explorer/id1455214828?mt=12) - this one works fine. It installs version 0.3.5 of MQTT Explorer.

The binary linked from the https://mqtt-explorer.com/ home page is different and does not start.
It [downloads](https://github.com/thomasnordquist/MQTT-Explorer/releases/download/v0.4.0-beta.6/MQTT-Explorer-0.4.0-beta.6-arm64.dmg) version 0.4.0-beta.6 and shows the above error.
I'm using macOS Sonoma 14.6.1 on an M1 Mac.

**Comment by bewinter** [comment]
Well, this is certainly not very helpful information to get rid of the error. This here isn't the anticipated solution either, but much closer to a real answer: https://github.com/thomasnordquist/MQTT-Explorer/issues/819#issuecomment-2257288817.
If you still want to run 0.4.0beta... :)
